### PR TITLE
uos make puppet-wechat great again

### PR DESCRIPTION
**Pull request by leochen-g**
uos 协议又可以重新使用了

### CODE REVIEW

**Comment by leochen-g** [comment]
有个lint 报错好像是项目之前就有的报错，是数据没有初始化，和本次提交无管

**Comment by LIUHanyang96** [comment]
关注一下

**Comment by lx0758** [comment]
mark

File: package.json [hunk 1]
```diff
@@ -61,6 +62,8 @@
     "@types/qr-image": "^3.2.5",
     "@types/request": "^2.48.8",
     "@types/xml2js": "^0.4.9",
+    "qrcode-terminal": "^0.12.0",
```
Comment (huan) [inline package.json:65]: Please remove unrelated dependencies.

File: src/puppet-wechat.ts [hunk 2]
```diff
@@ -82,6 +80,7 @@ type PuppetWeChatOptions = PUPPET.PuppetOptions & {
   launchOptions? : LaunchOptions
   stealthless?   : boolean
   uos?           : boolean
+  token?           : string
```
Comment (huan) [inline src/puppet-wechat.ts:83]: Could you please explain what's this `token` for? It seems that it does not been referenced in the new code.

**Review by huan — CHANGES_REQUESTED** [review]
Thanks for the fix!

Please follow my reviews, and also remember to make sure the CI turns green before we can continue reviewing this PR!

**Comment by huan** [comment]
LGTM

**Review by huan — APPROVED** [review]

**Pull request merged**